otx/cli/tools/train.py

```diff
@@ -222,11 +248,215 @@ def main():
     assert resultset.performance is not None
     print(resultset.performance)
 
-    if args.save_logs_to:
-        tmp_path = task.project_path
-        logs_path = os.path.join(args.save_logs_to, tmp_path.split("/")[-1])
-        shutil.copytree(tmp_path, logs_path)
-        print(f"Save logs: {logs_path}")
+    if args.gpus:
+        multigpu_manager.finalize()
+
+
+class MultiGPUManager:
+    """Class to manage multi-GPU training."""
+
+    def __init__(self, gpu_ids: str, multi_gpu_port: str):
+        self._gpu_ids = self.get_gpu_ids(gpu_ids)
+        self._multi_gpu_port = multi_gpu_port
+        self._main_pid = os.getpid()
+        self._processes: Optional[List[mp.Process]] = None
+
+    @staticmethod
+    def get_gpu_ids(gpus: str) -> List[int]:
+        """Get proper GPU indices form `--gpu` arguments.
+
+        Given `--gpus` argument, exclude inappropriate indices and transform to list of int format.
+
+        Args:
+            gpus (str): GPU indices to use. Format should be Comma-separated indices.
+
+        Returns:
+            List[int]:
+                list including proper GPU indices.
+        """
+        num_available_gpu = torch.cuda.device_count()
+        gpu_ids = []
+        for gpu_id in gpus.split(","):
+            if not gpu_id.isnumeric():
+                raise RuntimeError("--gpus argument should be numbers separated by ','.")
+            gpu_ids.append(int(gpu_id))
+
+        wrong_gpus = []
+        for gpu_idx in gpu_ids:
+            if gpu_idx >= num_available_gpu:
+                wrong_gpus.append(gpu_idx)
+
+        for wrong_gpu in wrong_gpus:
+            gpu_ids.remove(wrong_gpu)
+
+        if wrong_gpus:
+            print(f"Wrong gpu indeces are excluded. {','.join([str(val) for val in gpu_ids])} GPU will be used.")
```

Comment:
How about using logger's warning?

Reply:
yeah... actually logger isn't used previously, so I used also print. is it better to use logger instead print? how do you think about?

Reply:
If this is warning, then how about add [Warning] flag to the stdout

Reply:
I see. I'll modify code as you said.